.github/actions/oss-rules-of-play-action/entrypoint.sh

```diff
@@ -55,9 +62,11 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+# Cleanup
+rm -rf fosstars-rating-core > /dev/null 2>&1
+rm -rf .fosstars > /dev/null 2>&1
+
```

Comment:
Could this be affected by some folder permissions?

Reply:
Those directories are created by the action (by same process/user that tries to delete them), so there should not be issues with permissions.